copyIOSLibs.sh

```diff
@@ -1,5 +1,4 @@
 #!/bin/bash
-cp -f ./xamarin/FHXamarinIOSSDK/bin/Release/FHSDK.dll ./Dist/lib/xamarinios10/
-#cp -f ./xamarin/FHXamarinIOSSDK/bin/Release/FHSDK.xml ./Dist/lib/monotouch/
-cp -f ./xamarin/FHXamarinIOSSDK/bin/Release/FHXamarinIOSSDK.dll ./Dist/lib/xamarinios10/
-cp -f ./xamarin/FHXamarinIOSSDK/bin/Release/FHXamarinIOSSDK.XML ./Dist/lib/xamarinios10/
\ No newline at end of file
+cp -f ./xamarin/FHXamarinIOSSDK/bin/Debug/FHSDK.dll ./Dist/lib/xamarinios10/
```

Comment:
@corinnekrych nope it left over from when we moved xamarin into a sub directory. I've changed it.